FirebaseUI.podspec

```diff
@@ -64,7 +64,7 @@ Pod::Spec.new do |s|
                                 'Auth/FirebaseAuthUI/FUIAuthTableViewCell.h',
                                 'Auth/FirebaseAuthUI/FUIAuthTableHeaderView.h']
     auth.source_files = ['Auth/FirebaseAuthUI/**/*.{h,m}', 'Auth/FirebaseAuthUI/*.{h,m}']
-    auth.dependency 'Firebase/Auth', '~> 5.0'
+    auth.dependency 'Firebase/Auth', '~> 5.4'
```

Comment:
Need confirmation from @morganchen12

Reply:
yea. The error message seems to build against old Auth sdk, and not able to find new declarations of FIROAuthProvider